manywheel/build_cpu.sh

```diff
@@ -43,13 +43,10 @@ DEPS_SONAME=(
 
 rm -rf /usr/local/cuda*
 
-# builder/test.sh requires DESIRED_CUDA to know what tests to exclude
-export DESIRED_CUDA='cpu'
```

Comment:
What's the reasoning for removing this? Seems like we would still need this regardless

Reply:
`DESIRED_CUDA`,`DESIRED_PYTHON`,`PACKAGE_TYPE` are now persisted via `$BASH_ENV` from `binary_populate_env.sh` step. The reason it was set here is for the script invoked by this script. 
We can verify it by looking at circleci's output that `DESIRED_CUDA` has value `cpu`:
https://app.circleci.com/pipelines/github/pytorch/builder/1903/workflows/b2161f94-a657-479d-b7bf-d2462fc53f80/jobs/22228?invite=true#step-104-8815